typescript/cli/src/avs/config.ts

```diff
@@ -0,0 +1,19 @@
+import { ChainMap } from '@hyperlane-xyz/sdk';
+import { Address } from '@hyperlane-xyz/utils';
+
+interface AVSContracts {
+  avsDirectory: Address;
+  proxyAdmin: Address;
+  ecdsaStakeRegistry: Address;
+  hyperlaneServiceManager: Address;
+}
+
+// TODO: move to registry
```

Comment:
Will you be moving this to the registry before merging?

Reply:
no

Reply:
I was thinking we have a AVS deployment folder that houses the different challengers and this could live there too?